### DISCUSSION TITLE
在线视频有断流！！？？

### DISCUSSION THREAD

**Discussion by kitkw** [messense/aliyundrive-webdav#218] [Q&A] [answered]
### 问题描述

aliyundrive-webdav 1.1.0  + Rclone   挂载到 jellyfin 。jellyfin网页端  和  KODI客户端  都有断流！！！缓冲完一小段后看完了 卡在那里  等一小会才再继续缓冲继续看 。不停重复................

### 重现步骤

11

### 版本

aliyundrive-webdav 1.1.0

### 运行平台

openwrt

### 日志

11

**Comment by messense** [comment, marked as answer]
请提供日志。

**Reply by kitkw** [reply]
> 请提供日志。
日志是昨天的token信息。当下测试没有新日志生成，还是看一段停一下 的......................

**Reply by messense** [reply]
试试 kodi 直连不要经过rclone和jellfin，排除是它们导致的问题。

**Reply by kitkw** [reply]
> 试试 kodi 直连不要经过rclone和jellfin，排除是它们导致的问题。

好像是Rclone问题，用KODI直连就没事。阿里云盘好像没有API的吧？？要怎样解决这问题？？

**Reply by messense** [reply]
既然不是这个项目的问题，建议去给 rclone/jellfin 提 issue.
